pkg/debug/spec.sh

```diff
@@ -297,17 +366,66 @@ __EOT__
    fi
 done
 
-#enumerate NICs
+#enumerate cellular modems
+ID="0"
+for USBDEV in /sys/class/usbmisc/*; do
+    get_modem_protocol "$USBDEV" >/dev/null || continue
+    IFNAME="$(get_modem_ifname "$USBDEV")"
+    MODEMS="${MODEMS}${IFNAME}\n" # skip during NIC enumeration
```

Comment:
Is this comment visible?

Reply:
Somehow I missed this comment...
Interface names of multiple modems are separated by newline characters in the string.
grep searches for patterns within each line of text separately and does not consider newlines when matching.
I used this approach instead of arrays because Bourne shell does not support them and we do not have more advanced shell like Bash inside the debug container.

Reply:
got it.